src/librustdoc/html/static/rustdoc.css

```diff
@@ -1790,6 +1790,18 @@ details.rustdoc-toggle > summary::before {
 	cursor: pointer;
 }
 
+details.rustdoc-toggle.top-doc > summary,
+details.rustdoc-toggle.top-doc > summary::before,
+details.rustdoc-toggle.non-exhaustive > summary,
+details.rustdoc-toggle.non-exhaustive > summary::before {
+	font-family: 'Fira Sans';
+	font-size: 16px;
+}
```

Comment:
Why do we need to set the font size and family separately here? Is there not an appropriate existing rule to attach these to?

Reply:
Before that it was a "special case" for the `collapse-toggle`. So unless I missed it, I don't think there is an appropriate existing rule to attach it to.